docs/diagrams/ModelClassDiagram.puml

```diff
@@ -42,6 +47,11 @@ Client *--> Phone
 Client *--> Email
 Client *--> Address
 Client *--> "*" Tag
+Client *--> InsurancePlanManager
+InsurancePlanManager *--> "*" InsurancePlan
+InsurancePlanManager *--> "*" Claim
+BasicPlan --|> InsurancePlan
+TravelPlan --|> InsurancePlan
```

Comment:
Probably can make it go upwards to make the diagram easier to understand like:

```puml
BasicPlan --.up.|> InsurancePlan
TravelPlan --.up.|> InsurancePlan
```

Reply:
Thanks for the suggestion, that's a good way to format our UML diagram!